README.md

```diff
@@ -77,10 +81,19 @@ The file `configs/config.yaml` contains all information about used parameter val
 
 Some entries require your editing while most of them you can leave unchanged. The comments should give you the information about the meaning of the individual parameters. If you need to change path names please ensure to **use relative instead of absolute path names**.
 
+### 4. Prepare a "samples.tsv"
+This file will contain the names (column header "ID"), conditions (header "condition") and paths (relative to the execution directory)(header "bam") to all your input bam files. For an example see [configs/samples.tsv][sample-tsv]
+> NOTE: PAQR requires `.bam` AND corresponding `.bam.bai` files to be placed alongside in the same directory. It also expects the basenames of the two files to be the same. Thus, only .bam filepaths have to be given in the samples table, and the corresponding .bai filepath is inferred from there.
 
 
 ## Execution
-Create a new directory for your analysis within this directory and cd into it. Make sure you have the conda environment `paqr2` activated. For your convenience, the directory `execution` contains bash scripts that can be used to start local and slurm runs, using either singularity or conda. In order for the latter to work, you will have to specify the partition to be used in the respective bash scripts at `-p [PARTITION]`.
+Create a new directory for your analysis within this directory and cd into it. Make sure you have the conda environment `paqr2` activated. For your convenience, the directory `execute` contains bash scripts that can be used to start local and slurm runs, using either singularity or conda.
+
+For example, you could run the example config `configs/config.yml` locally with singularity with:
+
+```bash
+bash snakemake_local_run_singularity_containers.sh configs/config.yml
+```
```

Comment:
We can leave it like this for now but I want to give you a short heads-up.
Last week I have updated my PAQR version I have in MAPP such that I have added snakemake profiles and now there is just one main bash script to call the pipeline. Depending on the command line arguments to that `run.sh` script a different snakemake profile will be run. Once MAPP is public I can upgrade this version as well.
But FYI, we are asynchronous again. I needed to incorporate these recent changes in order to prepare MAPP to be easily run by the reviewers.